apps/web/components/booking/BookingListItem.tsx

```diff
@@ -526,6 +526,7 @@ function BookingListItem(booking: BookingItemProps) {
             </>
           ) : null}
           {isBookingInPast && isPending && !isConfirmed ? <TableActions actions={bookedActions} /> : null}
+          {isBookingInPast && isConfirmed ? <TableActions actions={bookedActions} /> : null}
```

Comment:
Can you add some space between 'Edit' and 'Check for recordings'
<img width="450" alt="Screenshot 2024-08-08 at 9 30 58 AM" src="https://github.com/user-attachments/assets/317cf959-e2a2-420b-b90b-d3784b030a86">

Also, mobile view ist broken
<img width="347" alt="Screenshot 2024-08-08 at 9 30 20 AM" src="https://github.com/user-attachments/assets/63238e9a-308b-4d37-9f1e-303bc4b41e70">